configurations/batch/batch.ini.template

```diff
@@ -544,6 +544,7 @@ type												= KAsyncDropFolderWatcher
 maximumExecutionTime								= 3000
 scriptPath											= ../plugins/drop_folder/batch/DropFolderWatcher/KAsyncDropFolderWatcherExe.php
 params.tags											= *
+params.mrss.xmlPath									= @WEB_DIR@/tmp/dropFolderFiles
```

Comment:
Since the kMrssXmlHandler later needs to use this path to create a bulk upload job (moved this file from this location to create a filesync), the location has to be in the @WEB_DIR@, otherwise the code has no permission to use the file (as with the import job).